lib/ansible/modules/cloud/amazon/redshift.py

```diff
@@ -422,6 +446,8 @@ def main():
         cluster_type                        = dict(choices=['multi-node', 'single-node', ], default='single-node'),
         cluster_security_groups             = dict(aliases=['security_groups'], type='list'),
         vpc_security_group_ids              = dict(aliases=['vpc_security_groups'], type='list'),
+        skip_final_cluster_snapshot         = dict(aliases=['skip_snapshot'], type='bool', default=False),
+        final_cluster_snapshot_identifier   = dict(aliases=['snapshot_identifier'], required=False),
```

Comment:
I think this "snapshot_identifier" alias might be too terse. How about "final_snapshot_id" or similar?